hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/snapshot/ExportSnapshot.java

```diff
@@ -538,21 +537,47 @@ private FileChecksum getFileChecksum(final FileSystem fs, final Path path) {
      * Check if the two files are equal by looking at the file length, and at the checksum (if user
      * has specified the verifyChecksum flag).
      */
-    private boolean sameFile(final FileStatus inputStat, final FileStatus outputStat) {
+    private boolean sameFile(final FileStatus inputStat, final FileStatus outputStat,
+      final boolean force) throws IOException {
```

Comment:
The parameter name 'force' seems a bit confusing here...

Maybe a better choice is to introduce another method called verifyCopyResult?

Reply:
OK sir,  I will have a new commit soon.